src/components/ServerSdkContext.tsx

```diff
@@ -44,7 +44,21 @@ export const ServerSDKProvider = ({
       }),
     [serverToken],
   );
+  const httpServerUrl = signalingProtocol + "://" + signalingHost + signalingPath.replace("peer", "server");
+  const serverWebsocket = useMemo(() => (httpServerUrl ? new WebSocket(httpServerUrl) : null), [httpServerUrl]);
+  if (serverWebsocket) {
+    serverWebsocket.binaryType = "arraybuffer";
+    // create a new writer
+    const authRequestWriter = protobuf.Writer.create();
+    const subscribeRequestWriter = protobuf.Writer.create();
+    const buffer = authRequestWriter.uint32(58).fork().uint32(10).string("development").ldelim().finish();
+    const subscr = subscribeRequestWriter.uint32(66).fork().uint32(8).int32(1).ldelim().finish();
```

Comment:
Can we use a more user-friendly version of this code? You mentioned that that is an inline version of some other method

Reply:
I like the crazy random numbers though :///
Fixed nonetheless